broker-daemon/orderbook/index.spec.js

```diff
@@ -312,9 +319,26 @@ describe('Orderbook', () => {
         expect(watcher.removeListener).to.have.been.calledWith('error', sinon.match.func)
       })
 
-      it('sets a 5 second timeout', () => {
+      it('exponentially backs off after repeated end events', () => {
```

Comment:
you should add some tests exercising each of the methods that you added

Reply:
Those additional methods have been removed after refactoring.

I'm not stoked about being unable to stub the new `watchMarketCall` due to its scope, so re-initialization is now tested by resetting the stub history for `watcher.once` and making sure it sets new listeners after the timeout. Let me know if you see a better approach to testing this.